flask_project/campaign_manager/reminderRedisClient.py

```diff
@@ -0,0 +1,11 @@
+from redis import Redis
+
+r_server = Redis("localhost")  # change localhost to IP
+
+
+def store_message(id_message, id_sender, reminder, reminder_time, delivered):
```

Comment:
what does this function do? I didnt find any reference to this function in the codebase.
Why do we need Redis?

Reply:
A future feature for the field campaigner would be that the admin(campaign creator) would be able to set up reminders for the campaign, which would be delivered to all the members involved in the particular campaign. We decided to use redis and celery for this feature. It is yet to be implemented.